pandas/_libs/tslibs/parsing.pyx

```diff
@@ -0,0 +1,681 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+# cython: linetrace=False
+# distutils: define_macros=CYTHON_TRACE=0
+# distutils: define_macros=CYTHON_TRACE_NOGIL=0
```

Comment:
I suppose those were for debugging? (if set to True) 
We don't keep those anywhere else (apart from the profile=False), so not sure we should do it here

Reply:
I'll be happy to remove these if requested. I habitually keep them around so I don't have to remember what the options are.  Say the word and they're gone.